test/langtools/tools/javac/lambdaShapes/org/openjdk/tests/vm/FDSeparateCompilationTest.java

```diff
@@ -21,18 +21,12 @@
  * questions.
  */
 
```

Comment:
```suggestion
*/

// this test has been disabled because of timeout issues.
// see JDK-8006746

```

Restore comment lines as their removal is not the task of this change.

Reply:
<img width="529" height="120" alt="image" src="https://github.com/user-attachments/assets/dfbd9842-bc99-456e-95b0-becbe2e0c1cf" />